pyproject.toml

```diff
@@ -18,25 +18,25 @@ rpaframework-recognition = "^5.2.0"
 rpaframework-windows = "^7.4.0"
 
 [tool.poetry.group.dev.dependencies]
-robotframeworklexer = "^1.1"
-sphinx = "^5.3.0"
-sphinx-rtd-theme = "^1.1.1"
-jinja2 = "^3.1.2"
-markupsafe = "^2.1.1"
-toml = "^0.10.2"
-sphinx-markdown-builder = "^0.5.4"
-robotframework-docgen = "^0.16.0"
-pylint = "^2.4.4, <2.13"
-sphinx-issues = "^3.0.1"
-docutils = "^0.16"
-black = "^22.3.0"
 invoke = "^1.6.0"
-colorama = "^0.4.5"
-keyring = "^23.9.0"
+toml = "^0.10.2"
 PyYAML = ">=5.4.1,<7.0.0"
+keyring = "^23.9.0"
+colorama = "^0.4.5"
+black = "^22.3.0"
+pylint = "^2.4.4, <2.13"
 flake8-docstrings = "^1.6.0"
 flake8-rst-docstrings = "^0.2.7"
+sphinx = "^5.3.0"
+sphinx-rtd-theme = "^1.1.1"
+sphinx-markdown-builder = "^0.6.5"
+sphinx-issues = "^3.0.1"
 sphinx-lint = "^0.6.6"
+robotframework-docgen = "^0.15.0"
```

Comment:
Any specific reason why not to use the latest 0.16.0 ?

Reply:
I think might have been a merge/rebase oversight! I'll bump it